farm/modeling/tokenization.py

```diff
@@ -567,7 +567,7 @@ def tokenize_batch_question_answering(pre_baskets, tokenizer, indices):
     baskets = []
     # # Tokenize texts in batch mode
     texts = [d["context"] for d in pre_baskets]
-    tokenized_docs_batch = tokenizer.batch_encode_plus(texts, return_offsets_mapping=True, return_special_tokens_mask=True, add_special_tokens=False)
+    tokenized_docs_batch = tokenizer.batch_encode_plus(texts, return_offsets_mapping=True, return_special_tokens_mask=True, add_special_tokens=False, verbose=False)
```

Comment:
Isnt there a parameter that controls padding explicitly? 
When this parameter is set to false it should not print the warning. When it still prints the warning this might be a bug in transformers (you can open an issue there once you have checked that for the current version of transformer + tokenizer)

Reply:
Seems like wanted logging behavior by transformers.